docs/protocol/10-DataSchemes/01-organization-metadata.md

```diff
@@ -88,7 +84,7 @@ The body of the POST request submitted to `url` will contain a JSON body like:
   "request": {
     "method": "register",
     "actionKey": "sign-up",
-    "entityId": "0xaabbccdd...",
+    "OrganizationId": "0xaabbccdd...",
```

Comment:
Are you sure is "OrganizationId" with the first uppercase letter?

Reply:
True! I force pushed a fix